asabiris/output/mattermost/service.py

```
@@ -79,8 +79,8 @@ def _resolve_config(self, tenant=None):
 		if tenant and self.ConfigService is not None:
 			try:
 				tenant_config = self.ConfigService.get_mattermost_config(tenant)
-			except KeyError:
-				L.warning("Tenant-specific Mattermost configuration not found for '%s'. Using global config.", tenant)
+			except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)